packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -231,8 +233,10 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
         };
 
         // Reference to this client supporting get_ops flow.
-        // back-compat: remove cast to any once new definition of IConnect comes through.
-        (connectMessage as any).supportedFeatures = { [feature_get_ops]: true };
+        connectMessage.supportedFeatures = { };
+        if (mc.config.getBoolean("Fluid.Driver.Odsp.GetOpsEnabled") === true) {
```

Comment:
I'd probably rename "GetOpsEnabled" to DisableInitialOps or something like that, as it does not disable client behavior, it only changes what client advertises that changes server behavior.

Reply:
I prefer it to map to what the config controls, rather than the side effects of that